web/src/routes/SurveyPages/SurveyLoginReqPage/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function SurveyLoginReqPage() {
```

Comment:
Same above. Use full name on every component.

This could be confusing

- Survey Login Required Page
- Survey Login Requested Page